### ISSUE TITLE
Check Chinese name tags in tiles

### ISSUE THREAD

**Issue by russss** [openinframap/openinframap#195]
[At the moment](https://github.com/openinframap/openinframap/blob/fcaf31da19973236a3faf30ad2c8dfab41aa1d11/tegola/layers.yml#L47-L48) we're only using the `name:zh` tag, we probably also need to look at `name:zh-Hant` and `name:zh-Hans` (although `name:zh` is the most popular).

**Comment by SteveLz** [comment]
Of course, for users, only `name:zh-Hans` and `name:zh-Hant` are needed, and `name:zh` is not a useful option because it will mix the two, but `name:zh` should be used as a fallback for both.
